src/tools/dev/scripts/bv_support/bv_qt.sh

```diff
@@ -248,6 +248,15 @@ function apply_qt_patch
                 fi
             fi
         fi
+
+        if [[ "$OPSYS" == "Linux" ]]; then
+            if [[ "$C_COMPILER" == "gcc" ]]; then
+                apply_qt_5101_gcc_9_2_patch
+                if [[ $? != 0 ]] ; then
+                    return 1
+                fi
+            fi
+        fi
```

Comment:
@brugger1 you didn't need to add another check for Linux, you could've added the compiler check to the linux conditional a few lines up.

Reply:
I moved the check up a few lines as suggested.